browser/components/RealtimeNotification.js

```diff
@@ -1,4 +1,5 @@
-import React, { PropTypes } from 'react'
+import PropTypes from 'prop-types'
```

Comment:
@yosmoc 
Please resolve conflict.
This line is not needed, I think.

Reply:
fixed!